src/Chocolatey-AU.psd1

```diff
@@ -0,0 +1,127 @@
+#
+# Generated on: 3/2/2024
+#
+
+@{
+
+    # Script module or binary module file associated with this manifest.
+    RootModule        = 'Chocolatey-AU.psm1'
+
+    # Version number of this module.
+    ModuleVersion     = '0.1.0'
+
+    # Supported PSEditions
+    # CompatiblePSEditions = @()
+
+    # ID used to uniquely identify this module
+    GUID              = 'b2cb6770-ecc4-4a51-a57a-3a34654a0938'
+
+    # Author of this module
+    Author            = 'Miodrag Milic, Chocolatey Community'
+
+    # Company or vendor of this module
+    CompanyName       = 'Chocolatey Software, Inc.'
+
+    # Copyright statement for this module
+    Copyright         = 'Miodrag Milic, Chocolatey Software, Inc.'
+
+    # Description of the functionality provided by this module
+    Description       = 'Chocolatey Automatic Package Updater Module'
+
+    # Minimum version of the Windows PowerShell engine required by this module
+    PowerShellVersion = '5.0'
+
+    # Name of the Windows PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the Windows PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # CLRVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = ''
+
+    # Modules that must be imported into the global environment prior to importing this module
+    # RequiredModules = @()
+
+    # Assemblies that must be loaded prior to importing this module
+    # RequiredAssemblies = @()
+
+    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
+    # ScriptsToProcess = @()
+
+    # Type files (.ps1xml) to be loaded when importing this module
+    # TypesToProcess = @()
+
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    # NestedModules = @()
+
+    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+    FunctionsToExport = 'Get-AUPackages', 'Get-RemoteChecksum', 'Get-RemoteFiles',
+    'Get-Version', 'Push-Package', 'Set-DescriptionFromReadme',
+    'Test-Package', 'Update-AUPackages', 'Update-Package'
+
+    # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+    CmdletsToExport   = '*'
+
+    # Variables to export from this module
+    VariablesToExport = '*'

```

Comment:
Again, lets not use a wildcard but rather specify it explicitly either manually or automatically.

Reply:
👍